packages/vite-plugin-cloudflare/playground/cloudflare-env/vite.config.custom-mode.ts

```diff
@@ -3,5 +3,5 @@ import { defineConfig } from "vite";
 
 export default defineConfig({
 	mode: "custom-mode",
-	plugins: [cloudflare({ persistState: false })],
+	plugins: [cloudflare({ inspectorPort: 0, persistState: false })],
```

Comment:
What effect does setting this to 0 have?
Does it ask the plugin to just choose its own free port, or does it disable debugging altogether.
If the former, is there a mechanism for disabling the inspector?

Reply:
Now we accept `false` too and I've updated the playground apps to use that 🙂👍